pkgs/by-name/nr/nrr/package.nix

```diff
@@ -5,12 +5,14 @@
 , darwin
 , pkg-config
 , libiconv
+, lto ? true
```

Comment:
The convention is to have some prefix such as `withLTO` or `enableLTO` to make it clear that this is a boolean flag, not a package.

It also avoids cases where there might be a package called `lto` in the future which would be put in here by `callPackage`.

Reply:
Renamed the option to `enableLTO`, thanks for the heads up!